UserCoreModel.php

```diff
@@ -0,0 +1,1732 @@
+<?php
```

Comment:
Oops, looks like `UserCoreModel.php` has been added in the root directory.

Reply:
Oh, sorry about that! I'm not yet very familiar with Github.
I deleted the file in the root. The first committed **UserCoreModel.php** should be at the right position. Only the function **readProfileIdOf** has been added.

Reply:
I've seen that there has been already changes to the UserCoreModel file, so I downloaded the latest version, added the function and uploaded it again. In the hurry I must have made a mistake with the location.

Reply:
@dimuller Hi !
What you shall do is for modifying in Github
1. clone repo.
2. create a branch from master
3. make modification.
4. make a git pull/fetch to update your repo
5. merge back to master
6. push back and make the PR

Reply:
@dimuller Also, I think people shall be allowed to vote on their own profile, many time time also. So they can have higher rating and feel more confident of themselves.

Reply:
@polynamaude thanks for you guidelines. On the next occasion I will try to follow your instructions.